internal/config/config.go

```diff
@@ -123,12 +124,19 @@ func Get() *IngressConfig {
 		options.SetDefault("KafkaBrokers", clowder.KafkaServers)
 		options.SetDefault("KafkaTrackerTopic", clowder.KafkaTopics["platform.payload-status"].Name)
 		options.SetDefault("KafkaAnnounceTopic", clowder.KafkaTopics["platform.upload.announce"].Name)
+
+		if *broker.SecurityProtocol != "" {
+			options.Set("KafkaSecurityProtocol", *broker.SecurityProtocol)
+		} else if *broker.Sasl.SecurityProtocol != "" {
+			options.Set("KafkaSecurityProtocol", *broker.Sasl.SecurityProtocol)
+		} 
```

Comment:
```suggestion
		if brorker.SecurityProtocol != nil && *broker.SecurityProtocol != "" {
			options.Set("KafkaSecurityProtocol", *broker.SecurityProtocol)
		} else if broker.Sasl != nil && broker.Sasl.SecurityProtocol != nil && *broker.Sasl.SecurityProtocol != "" {
			options.Set("KafkaSecurityProtocol", *broker.Sasl.SecurityProtocol)
		} 
```

Reply:
Belts and suspenders for me!!  ;)

Reply:
I'm really not sure what the underlying config is going to look like for each env.  So I'm being overly cautious...